README.md

```diff
@@ -1,7 +1,17 @@
-# MapLibre GL style specification & utilities - Design proposals are required
+# MapLibre GL style specification & utilities
 
 This directory contains code and reference files that define the MapLibre GL style specification and provides some utilities for working with MapLibre styles.
 
+The style specification is used in MapLibre GL JS and in MapLibre GL Native. Our long-term goal is to have feature parity between the web and the native libraries.
+
+## Contributing
+
+If you want to contribute to the style specification, please open an issue with a design proposal. Once your design proposal has been accepted, you can open a pull request and implement your changes.
```

Comment:
I would consider changing the new issue template to support this and either keep it here too or remove it from here.

Reply:
#72